Lesson 5 - CSS/code/exercises/exercise 1/theme.css

```diff
@@ -1,3 +1,108 @@
-.row {
-    display: block;
+* {
+	margin: 0 px;
+	padding: 0 px;
+}
+
+
+#first{
+	float:left;
+	margin-left:2%;
+	border-top-left-radius: 5%;
+	border-top-right-radius: 5%;
+	border-bottom-left-radius: 5%;
+	border-bottom-right-radius: 5%;
+	width: 45%;
+	background-color:#D3D3D3;
+	overflow:hidden;
+	
+}
+#first h1 {
+	width:100%;
+	height:80px;
+	color: white;
+	background-color:black;
+	line-height:80px;
+	padding: 0 15px;
+	margin: 0;
+	
+}
+
+#first p{
+	margin-left:5%;
+	margin-right:5%;
+}
+#first p:first-letter
+{
+		color:black;
+		font-size:50px;
+		
+}
+
+#second{
+	float:right;
+	margin-right:2%;
+	border:5px solid;
+	width: 45%;
+	background-color:green;
+	box-shadow: 12px 12px 20px
+}
+
+#second h1 {
+	margin: 0;
+	text-indent:15px;
+	font-style:italic;
+	width:100%;
+	height:80px;
+	color: white;
+	background-color:#FF8C00;
+	line-height:80px;
+	
+}
+
+#second p{
+	margin-top:5%;
+	margin-left:5%;
+	margin-right:5%;
+	color:white;
+}
+#second p:first-line
+{
+		color:white;
+		font-weight:bold;
+		font-size:20px;
+		text-shadow:-5px 5px 5px white;
+		
+}
+
+
+
+#third {
+	float:left;
+	margin-left:15%;
+	margin-right:15%;
+	margin-top:5%;
+	
+}
+
+#third  h1{
+	width: 15%;
+    font-size:30px;
+    font-style:bold;
+    text-align:center;
+}
+
+#third p {
+    width:14%;
+    white-space:nowrap;
+    overflow:hidden;
+    text-overflow:ellipsis;
+  
+    
+}
+.obraz {

```

Comment:
Do not mix polish and english selector names :-1:. Be professional. Drop polish. Please.